integration_test/ops_agent_test.go

```diff
@@ -356,13 +357,13 @@ func TestProcessorOrder(t *testing.T) {
         receivers: [mylog_source]
         processors: [json1, json2]
         exporters: [google]
-`, logPath)
+`, logPath, "%Y-%m-%dT%H:%M:%S.%L%z")
 
 		if err := setupOpsAgent(ctx, logger, vm, config); err != nil {
 			t.Fatal(err)
 		}
 
-		line := fmt.Sprintf(`{"log":"{\"level\":\"info\",\"message\":\"start\"}\n","time":"%s"}`, time.Now().Format(time.RFC3339Nano)) + "\n"
+		line := fmt.Sprintf(`{"log":"{\"level\":\"info\",\"message\":\"start\"}\n","time":"%s"}`, time.Now().UTC().Format(time.RFC3339Nano)) + "\n"
```

Comment:
When not using UTC time stamps (e.g. 2022-02-10T08:07:29.018896832-08:00) the parsing fails in windows (b/218888265). This is one way to avoid the error until the bug is fixed, another way could be skipping the test for windows.

Reply:
I thinkn this deserves a comment in the code, otherwise the next person is going to have no idea why this is like that, and especially why it is different from the other test that does something similar without UTC and with Z.

Reply:
I found out that [TestCustomLogFormat](https://github.com/GoogleCloudPlatform/ops-agent/blob/bb8a03e38c2c71d2c5261a8e148ed6b4380c4e05/integration_test/ops_agent_test.go#L279) also throws the error

```
[error] [parser] cannot parse '2022-02-09T17:59:21.323823326Z' after %L
[ warn] [parser:my_pipeline.mylog_source.0] invalid time format %Y-%m-%dT%H:%M:%S.%L%Z for '2022-02-09T17:59:21.323823326Z'
```

The difference is that `parse_regex` sends the log with a "current time" timestamp (this is expected [out stackdriver plugin](https://github.com/fluent/fluent-bit/blob/1fa0e94a09e4155f8a6d8a0efe36a5668cdc074e/plugins/out_stackdriver/stackdriver.c#L2120)) and `parse_json` doesn't in this case. I've tested this (b/219090004) and seems to happen only with "%Z".

I could update both tests in another PR to use "%z", but it may need also to use UTC.

Reply:
Update : Added comment in code explaining difference with test TestCustomLogFormat.